meetings/2024-05-08.md

```diff
@@ -0,0 +1,141 @@
+* SC: It also conforms to / be compatible with some of the [W3C publication rules](https://www.w3.org/pubrules/doc) (without interfering with CG Report Requirements), so it should be even workable (First Public) Working Draft or later if/when we get there.  I've already went through this publication process with specs in WGs.
+* SC: CG-DRAFT can be published at https://solidproject.org/TR/protocol , with a link to "This version" for the specific snapshot (i.e., Version 0.11.0, e.g., `https://solidproject.org/TR/2024/{YYYYMMDD}-protocol`).
+* SC: Once we get passed that publication, we can look into releasing CG-FINAL. I'd still like to encourage the Group to communicate with implementation experience, e.g., who is implementing or does not want to implement N3 Patch, ditto IRIs,..., which can be used to make another CG-DRAFT and/or towards the CG-FINAL.
+* SC: As for the Editor's Draft at https://solidproject.org/ED/protocol , that's also being updated to better clarify (e.g., in copyright, Status of this Document) that it is not a CG report and of course continue to not imply that W3C is endorsing it. The document will live on as is, and we can cut CG-DRAFT/FINAL(s) from it.
+* SC: Lastly, I will PR with some of the other documents listed under https://solidproject.org/TR/ along these lines.
+* IJ: We have the goal to ??? and avoid confusion with other documents. Two models. Model 1: any specification produced by a CG is subject to requirements for CG reports. A subclass is considered "not yet adopted" by the CG.. I see the use of ED, that might not be the best label. Benefit: IPR/authorship/ownership is clearly the CG and its participants. Model 2: these aren't CG reports until approved by the CG. Prior to that there areno markings/styling indicating that it's a CG report. Priot to that they should not be referred to as published CG reports. No W3C branding, just documents that will eventually be submitted. 
+* eP: Is the intention all current drafts are CG drafts? We have work item process that people can start repos but once moved to Solid org they become community drafts. In WG there is WD and ??? We could have something similar. there are impleentations following the draft at different levels/versions. 
+* IJ: Within CG reports there are two statuses: draft and final. Final is for improving patent policy coverage. If you have other labels you will have to educate the comunity on those. I have a slight preference for all being CG drafts until declared final. If these are all works of the CG they should be CG reports as opposed to external work. 
+* TBL: Solid GH org follows the Solid process. TR contains documents that are not necessarily ??? If we don't have the 'final' flag ??? 
+* IJ: I'd rather not entertain the proposal to map CG process to Solid Team? process. I don't want to change Solid process, just find a way to label it within the CG process. FINAL is optional, not required. Status within the CG system can be comunicated. 
```

Comment:
@ianbjacobs , hard to tell from the minutes on the exact discussion - I wasn't present - but to be clear, Solid CG has a charter, and the only process is the one mentioned at https://www.w3.org/community/solid/charter/#community-group-process :

>Anything in this charter that conflicts with requirements of the [Community and Business Group Process](https://www.w3.org/community/about/agreements/) is void.

>Unless otherwise stated, the Community Group will follow or aim to be compatible with the [W3C Process](https://www.w3.org/Consortium/Process/).

Reply:
At that point in the meeting my comment amounted to this:

* The Solid community has a process. For the purposes of the current discussion, I didn't want to dive into whether the Solid process should be adopted more generally by CGs. 
* I was also saying that I would also like to not disrupt the Solid process (which I am not familiar with).
* Rather I was hoping to focus on a solution that would enable the group to publish CG reports consistent with those requirements as well as clearly send signals to the Solid community consistent with that process.